pkgs/top-level/all-packages.nix

```diff
@@ -8589,6 +8591,8 @@ let
 
   alsa-firmware = callPackage ../os-specific/linux/alsa-firmware { };
 
+  alsaTools = callPackage ../os-specific/linux/alsa-tools { };
```

Comment:
no need to use camelCase for attributes anymore. If the package name is ```also-tools```, please use that name for the attribute.